demo/src/javascripts/components/home.js

```diff
@@ -1,16 +1,14 @@
+/* global window */
```

Comment:
Nit: I would probably have declared `requestAnimationFrame` and `cancelAnimationFrame` as globals instead of `window`. More similar to how we import symbols.